pkger/service.go

```diff
@@ -1083,9 +1167,16 @@ func (s *Service) addStackPkgState(ctx context.Context, stackID influxdb.ID, pkg
 	// else
 	//	  add stub pkg resource that indicates it should be deleted
 	for _, r := range stack.Resources {
-		updateFn := pkg.setObjectID
-		if !pkg.Contains(r.Kind, r.Name) {
-			updateFn = pkg.addObjectForRemoval
+		var mapper stateMapper = pkg
+		if r.Kind.is(KindBucket, KindLabel) {
```

Comment:
right now only 2 types are supported, buckets and labels. This will grow as more models are moved over to new model